src/main/java/com/MedicalSpecializiation.java

```diff
@@ -1,9 +1,31 @@
 package com;
 
+import java.util.Scanner;
+
 public enum MedicalSpecializiation {
 
     PATHOLOGY,
     CARDIOLOGY,
     NEUROLOGY,
-    UROLOGY
+    UROLOGY;
+
+    public static MedicalSpecializiation getMedicialSpecializiation() {
+        System.out.println("Enter Doctor's medicial specializiation (PATHOLOGY, CARDIOLOGY, UROLOGY, NEUROLOGY):");
+        String selectedMedicialSpecializiation = getStringParameter("Your choise:");
```

Comment:
Thai method should only return specialization. Please  extract printing message and getting parameter.